src/cldr/load.ts

```diff
@@ -20,20 +20,23 @@ declare const require: Require;
  * @private
  * Load the CLDR JSON files at the specified paths.
  *
- * @param {paths}
+ * @param require
+ * An optional contextual require that can be used to resolve relative paths.
+ *
+ * @param paths
  * The JSON paths.
  *
  * @return
  * A promise to the CLDR data for each path.
  */
-const getJson: (paths: string[]) => Promise<CldrData[]> = (function () {
+const getJson: (require: any, paths: string[]) => Promise<CldrData[]> = (function () {
```

Comment:
For this internal method, `require` is always passed in; it will either be a context require or the main `require`. That seemed easier than having `if (typeof require === 'function')` checks in two different places.